### PR TITLE
feat(collapse): added collapse and chevron up for tests

### PR DESCRIPTION
**Pull request by Graeimh**

### CODE REVIEW

File: web/components/vue/InputDropdown.vue [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+};
+
+const classList = [
+  "input__border-background",
+  "input__container-flex",
+  "input__label-style",
+  "dropdown",
+];
+
+const isBrowserMozilla = new RegExp("Firefox").test(navigator.userAgent.toString());
```
Comment (lcubeddu) [inline web/components/vue/InputDropdown.vue:121]: Check de browser agent c'est pas ouf 
https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection#javascript

**Review by NyraSama — DISMISSED** [review]
LGTM

File: web/components/vue/InputDropdown.vue (hunk 1 above)
Comment (Graeimh) [reply web/components/vue/InputDropdown.vue]: A fix has been made according to this suggestion, thank you for the assistance.

**Review by NyraSama — APPROVED** [review]
LGTM

**Review by lcubeddu — APPROVED** [review]
</img>

**Pull request merged**